templates/TIR-infra.html.tmpl

```diff
@@ -0,0 +1,273 @@
+                        <tr>
+                            <td class="lean">Git Commit:</td>
+                            <td class="content-wrappable">{{metadata.git.commit}}</td>
+                        </tr>
+                        <tr>
+                            <td class="lean">Git Tag:</td>
+                            <td class="content-wrappable">{{metadata.git.targetTag}}</td>
+                        </tr>
+                        <tr>
+                            <td class="lean">Git URL:</td>
+                            <td class="content-wrappable">{{metadata.git.url}}</td>
+                        </tr>
+                        <tr>
+                            <td class="lean">OpenShift Cluster API URL:</td>
+                            <td class="content-wrappable">{{metadata.openShift.apiUrl}}</td>
+                        </tr>
+                        <tr>
+                            <td class="lean">Created by Jenkins Job Name:</td>
+                            <td class="content-wrappable">{{metadata.jenkins.jobName}}</td>
+                        </tr>
+                        <tr>
+                            <td class="lean">Created by Jenkins Build Number:</td>
+                            <td class="content-wrappable">{{metadata.jenkins.buildNumber}}</td>
+                        </tr>
+                        <tr>
+                            <td class="lean">Target environment:</td>
+                            <td class="content-wrappable">{{metadata.buildParameter.targetEnvironment}}</td>
+                        </tr>
+                    </table>
+                </td>
+                <td id="logo" class="lean"></td>
+            </tr>
+        </table>
+
+        <div id="blueband">&nbsp;</div>
+
+        <h1>Technical Installation Report for '{{metadata.name}}'</h1>
+
+        <h2>Table of Contents</h2>
+        <ol class="table-of-contents">
+            <li><a href="#section_1">Introduction</a></li>
+            <li><a href="#section_2">Installation</a></li>
+            <li><a href="#section_3">Diagnostics and Testing</a></li>
+            <li><a href="#section_4">Conclusion Statement</a></li>
+            <li><a href="#section_5">Definitions and Abbreviations</a>
+                <ol>
+                    <li><a href="#section_5_1">Definitions</a></li>
+                    <li><a href="#section_5_2">Abbreviations</a></li>
+                </ol>
+            </li>
+            <li><a href="#section_6">Reference Documents</a></li>
+            <li><a href="#section_7">Document History</a></li>
+        </ol>
+    </div>
+
+    <div class="page">
+        <h2 id="section_1"><span>1</span>Introduction</h2>
+
+        <p>This document describes the installation of the infrastructure component <em>{{metadata.buildParameter.configItem}}</em>. The installation is based on the corresponding Technical Installation Plan (<em>TIP</em>) of Config Item: {{metadata.buildParameter.configItem}}.</p>
```

Comment:
I would remove the word 'infrastructure' from here. 
We can use this quickstarter to deploy application components to the cloud, so this word here is a problem. I am thinking about deploying data pipelines for example. And that is not an infrastructure component. 

FYI @metmajer  @frankjoas  @nichtraunzer